Packages/vcs/Lib/Canvas.py

```diff
@@ -860,7 +860,7 @@ def objecthelp(self, *arg):
     #                                                                           #
     ##########################################################################
     def __init__(self, gui=0, mode=1, pause_time=0,
-                 call_from_gui=0, size=None, backend="vtk", geometry=None):
+                 call_from_gui=0, size=None, backend="vtk", geometry=None, bg=None):
```

Comment:
`bg` makes it so you can call `open` without displaying the canvas. I use it in the test I added (down at the bottom), to make it so this test doesn't flash on the screen (which I know you hate :wink:). I believe I can also use it to fully background-ize a couple of the other tests I've added that flash on the screen.